### PR TITLE
region: stabilize interface

### PR DESCRIPTION
**Pull request by emersion**
References: https://github.com/swaywm/wlroots/issues/1008

### CODE REVIEW

File: include/wlr/types/wlr_region.h [hunk 1]
```diff
@@ -1,17 +1,31 @@
...
+ * - New functions
+ * - New struct members
+ * - New enum members
+ *
+ * Note that wlroots does not make an ABI compatibility promise - in the future,
+ * the layout and size of structs used by wlroots may change, requiring code
+ * depending on this header to be recompiled (but not edited).
+ *
+ * Breaking changes are announced by email and follow a 1-year deprecation
+ * schedule. Send an email to ~sircmpwn/wlroots-announce+subscribe@lists.sr.ht
```
Comment (bl4ckb0ne) [inline include/wlr/types/wlr_region.h:13]: Should we actively use that mailing list? It seems unused

Comment (fluix-dev) [reply include/wlr/types/wlr_region.h:13]: I'll put in my vote for a mailing list where release notes are copied to alongside the existing GitHub atom feed.

Comment (emersion) [reply include/wlr/types/wlr_region.h:13]: My take: https://github.com/swaywm/wlroots/pull/3016

Comment (fluix-dev) [reply include/wlr/types/wlr_region.h:13]: Seems fair. An atom feed is good enough.

**Review by bl4ckb0ne — APPROVED** [review]

**Comment by bl4ckb0ne** [comment]
Thanks!

**Pull request merged**